cypress/integration/LockedSelector.feature

```diff
@@ -0,0 +1,119 @@
+Feature: In the main page, use the LockedSelector to navigate
```

Comment:
Feature holds a general description of the feature that is tested. Is only for us developers to describe what we test in this feature. Basically, `cypress` encourages us to categorise the tests by features.